nsqadmin/options.go

```diff
@@ -59,3 +62,23 @@ func NewOptions() *Options {
 		AdminUsers:               []string{},
 	}
 }
+
+func (opts *Options) Resolve(flagSet *flag.FlagSet, cfg map[string]interface{}) error {
```

Comment:
We already have a way to coerce config data into appropriate option types, see:

https://github.com/nsqio/nsq/blob/master/apps/nsqd/options.go#L69

So we don't actually need this new method for any of the binaries.

Reply:
@mreiferson Do we need to migrate the Validate method to Resolve? Because nsqadmin and nsqlookupd have no corresponding validate methods.

Reply:
No, I think each binary should have a config file validation method rather than moving that responsibility into the package / options.